collective/jsonmigrator/blueprints/datafields.py

```diff
@@ -34,29 +36,22 @@ def __iter__(self):
                 yield item
                 continue
 
-            obj = self.context.unrestrictedTraverse(
-                item['_path'].lstrip('/'), None)
+            path = safe_unicode(item['_path'].lstrip('/')).encode('ascii')
```

Comment:
@sunew Is your intention here to purposely raise a UnicodeDecodeError, if the path is not ascii? Is it to ensure that the path is ascii?